sources/PTYSession.m

```diff
@@ -3098,6 +3100,11 @@ - (void)setBlend:(float)blendVal {
     [_textview setBlend:blendVal];
 }
 
+- (void)setDefaultBGAlphaOnly:(BOOL)value
```

Comment:
I don't generally use abbreviations (although you'll find some in very old code). Please name this `setTransparencyAffectsOnlyDefaultBackgroundColor:` (and make the corresponding change in various other places)